security/cert_manager_operator/cert-manager-authenticate-gcp.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-authenticate-gcp"]
+= Authenticating the {cert-manager-operator} with GCP Workload Identity
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-authenticate-gcp
+
+toc::[]
+
+You can authenticate the {cert-manager-operator} on the GCP Workload Identity cluster. You can configure the cloud credentials by using the `ccoctl` binary.
+
+include::modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc[leveloffset=+1]
```

Comment:
@swghosh , @xenolinux , after reading this doc, it seems that users will probably not know what the purpose of it for cert-manager, i.e. after configuration, what else then? We seem to need document above configuration will be used by cert-manager for requesting certificates using `cloudDNS`, i.e. "Create a clusterissuer using Google clouddns without specifying explicit credential" as details in https://issues.redhat.com/browse/OCPBUGS-9998?focusedId=22351582&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22351582 .